src/components/tabs/EventView.vue

```diff
@@ -105,7 +105,7 @@
                     v-for="event in schedule.currentSchedule.events"
                     :key="event.key"
                     class="mx-3 align-items-center justify-content-between "
-                    :class="event.selected && 'table-primary'"
+                    :class="event == currentSelectedEvent && 'table-primary'"
```

Comment:
This is a hack of the logical shortcut...
A better way to write this is to use the Vue provided conditional class syntax:
```diff
- :class="event == currentSelectedEvent && 'table-primary'"
+ :class="{'table-primary': event === currentSelectedEvent}"
```
Btw, we use strict equality (`===`) whenever we can